tools/cxbe/Main.cpp

```diff
@@ -29,7 +29,8 @@ int main(int argc, char *argv[])
 
     bool bRetail;
     uint32 dwTitleId = 0xFFFF0002;
-    uint32 dwRegions;
+    uint32 dwRegions = XBEIMAGE_GAME_REGION_NA | XBEIMAGE_GAME_REGION_JAPAN |
+                       XBEIMAGE_GAME_REGION_RESTOFWORLD | XBEIMAGE_GAME_REGION_MANUFACTURING;
```

Comment:
This probably belongs into the commit that introduced being able to specify the region flags.

Reply:
Yeah more leftovers from the incinerated commit